conf/cassandra.yaml

```diff
@@ -1692,12 +1692,26 @@ drop_compact_storage_enabled: false
 # Uncomment the startup checks and configure them appropriately to cover your needs.
 #
 #startup_checks:
-#  filesystem_ownership:
+# Verifies correct ownership of attached locations on disk at startup. See CASSANDRA-16879 for more details.
+#  check_filesystem_ownership:
 #    enabled: false
 #    ownership_token: "sometoken" # (overriden by "CassandraOwnershipToken" system property)
 #    ownership_filename: ".cassandra_fs_ownership" # (overriden by "cassandra.fs_ownership_filename")
-#  dc:
+# Prevents a node from starting if snitch's data center differs from previous data center.
+#  check_dc:
 #    enabled: true # (overriden by cassandra.ignore_dc system property)
-#  rack:
+# Prevents a node from starting if snitch's rack differs from previous rack.
+#  check_rack:
 #    enabled: true # (overriden by cassandra.ignore_rack system property)
-
+# Upon node's start, checks that gcGraceSeconds of each user table added to timestamp in the heartbeat_file is lower
+# than the current time. This check prevents the node from starting with too old data where it might introduce data
+# inconsistency issue like resurrecting the data etc. Once the check passes, it will start to write a timestamp into
+# thr heartbeat_file every minute as long as the node is not drained. Heartbeat file will not be updated if this check
+# is disabled. If this check is enabled when no heartbeat file exists, no check against tables will be done and
+# timestamp will be written to heartbeat file every minute once all startup checks pass. On the successful check against
+# the heartbeat_file, the content of the file will be rewritten with new timestamp periodically.
+#  check_data_resurrection:
+#    enabled: false
+#    heartbeat_file: cassandra-heartbeat # file where Cassandra periodically writes the last time it was known to run
```

Comment:
Since this is disabled by default, we should use /var/lib/cassandra/data/cassandra-heartbeat for the example

Reply:
+1